packages/iris-grid/src/sidebar/VisibilityOrderingBuilder.tsx

```diff
@@ -883,9 +883,8 @@ class VisibilityOrderingBuilder extends Component<
         }}
       >
         <div className="top-menu">
-          <button
-            type="button"
-            className="btn btn-link"
+          <Button
```

Comment:
Something looks messed up in this top menu, the Toggle All button is wrapping for some reason:
![image](https://user-images.githubusercontent.com/4505624/196718404-88257a04-c60a-46ab-ad9a-8bba32f9b537.png)
Should look more like:
![image](https://user-images.githubusercontent.com/4505624/196718462-d3d7f70b-b274-42be-aba6-9ef6ae8b5e72.png)

Reply:
Looks like kind="ghost" adds the class `btn-link-icon-only` which doubles the padding on the 6 icon only buttons in the div.  This causes the rightmost one to just barely overflow and Toggle All matches the new height as a result.

Reply:
Might need to add some css to reduce the padding on these buttons then.